python/test/test_junit.py

```diff
@@ -7,6 +9,7 @@
 from publish.junit import parse_junit_xml_files, get_results, get_result, get_content, get_message, Disabled
 from publish.unittestresults import ParsedUnitTestResults, UnitTestCase, ParseError
 
+test_files_path = pathlib.Path(os.path.dirname(__file__)) / 'files'
```

Comment:
more like this, also in `python/test/test_publish.py`:

```suggestion
test_files_path = pathlib.Path(__file__).parent / 'files'
```